metrics/Code_Changes.md

```diff
@@ -70,7 +70,7 @@ Mandatory:
     those which do not touch files, and in some cases are not considered as
     reflecting a coding activity.
 
-## Use Cases
+## Objectives
```

Comment:
I'm not sure about "objectives". But I agree that captures the idea better than "use cases", and causes less confusion. So, if cannot find some other word, that's ok with me. Thanks!